paddle/fluid/operators/distributed_ops/lookup_sparse_table_merge_op.h

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License. */
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "paddle/fluid/framework/op_registry.h"
+
+namespace paddle {
+namespace operators {
+
+int64_t GetDelimiterForShard(const std::vector<int64_t>& rows, int start_idx,
+                             int shard_id, int shard_num) {
+  int64_t rows_num = rows.size() / 2;
```

Comment:
why divides 2? And it's inefficient for line 27~30.

Reply:
unused method， will be deleted.